### DISCUSSION TITLE
Theme fontBase and fontCode not working properly

### DISCUSSION THREAD

**Discussion by felipedwork** [storybookjs/storybook#24703] [Help]
Hey folks!

I am using storybook `v7.5.2` with custom `fontBase` and `fontCode` properties on the theme file.

![image](https://github.com/storybookjs/storybook/assets/142022838/5e731264-a354-4f01-b50b-5956e98de772)

They WORK on the side nav:

![image](https://github.com/storybookjs/storybook/assets/142022838/2cf3002a-6dd5-4d99-8d68-27e424888f2b)

They DON'T work on docs/canvas (mdx):
No matter what I add to those theme variables, the CSS of storybook is always override with specific selectors:

![image](https://github.com/storybookjs/storybook/assets/142022838/7e9cf7d1-fbc3-46eb-879b-5d6b474b9ab4)

**Comment by felipedwork** [comment]
Not sure if I can do this, but I am adding the comment to BUMP the discussion.

**Comment by niktariy** [comment]
Still not working.

Storybook version `8.4.7`

![image](https://github.com/user-attachments/assets/d6a65fc8-3baa-4c37-a73a-9d30fbad27e0)
![image](https://github.com/user-attachments/assets/622a8278-c837-4bc5-bd32-c1e88f0afd36)
![image](https://github.com/user-attachments/assets/159a061d-26bc-4e52-98bd-d1286ffe964a)
![image](https://github.com/user-attachments/assets/699bba54-f8bf-40a0-9a0c-0b672252cd03)
